packages/fiori/src/DynamicPageTitle.ts

```diff
@@ -212,6 +231,12 @@ class DynamicPageTitle extends UI5Element {
 
 	onBeforeRendering() {
 		this.prepareLayoutActions();
+
+		if (this.hasSnappedTitleOnMobile) {
```

Comment:
I noticed we don't use such approach throughout the project.

I think we should use a private property like this

	/**
	 * ---------------------------------.
	 * @private
	 */
	@property({ type: Boolean, noAttribute: true })
	_hasSnappedTitleOnMobile = false;

And if you think the decision for the prop should be on before rendering, then the line should look like this:

		this._hasSnappedTitleOnMobile = isPhone() && this.snapped && this.snappedTitleOnMobile.length > 0;

At the end you change it to the CSS to:
host([snapped][_hasSnappedTitleOnMobile])

By the way you check for snapped in the _hasSnappedTitleOnMobile, therefore the CSS can also be:
host([_hasSnappedTitleOnMobile])

Reply:
Resolved. :)